src/pages/docs/platform/auth/authentication.mdx

```diff
@@ -0,0 +1,1062 @@
+---
+title: Authentication Methods
+meta_description: "Comprehensive guide to Ably's authentication methods including key authentication, token authentication, JWT, and implementation patterns."
+redirect_from:
+  - /docs/auth/basic
+  - /docs/auth/token
+---
+
+Ably provides multiple authentication methods designed for different use cases and security requirements. This page covers the implementation details for each method and when to use them.
+
+## How to choose authentication method?
```

Comment:
I'd go with acitve voice titles as we try to normally (or so i think we do). So "choose an anth method etc.." would be more in line :)